### PR TITLE
Display list of routes using React

### PR DESCRIPTION
**Pull request by arkadyan**
[Convert /ladder page to React](https://app.asana.com/0/1112935048846093/1117328247505291)

### CODE REVIEW

File: assets/src/api.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      if (response.status !== 200) {
+        throw new Error(`Response error: ${response.status}`)
+      }
+      return response
+    })
+    .then((response: Response) => response.json())
+    .then(({ data: routes }: RoutesRequestResponse) => routes)
+    .catch(error => {
+      // tslint:disable-next-line: no-console
+      console.log(`Error loading routes`, error)
```
Comment (paulswartz) [inline assets/src/api.ts]: `console.error`?

**Comment by arkadyan** [comment]
@meagonqz if you have a free moment to take a look at this and see if you have any general React advice, we'd appreciate it.

**Comment by arkadyan** [comment]
@paulswartz good suggestion, I made a ticket for channels here: https://app.asana.com/0/1112935048846093/1117706293711434

**Comment by arkadyan** [comment]
Updated the file names

File: assets/package.json [hunk 2]
```diff
@@ -27,14 +28,17 @@
     "@types/phoenix": "^1.4.0",
     "@types/react": "^16.8.8",
     "@types/react-dom": "^16.8.2",
+    "@types/react-test-renderer": "^16.8.1",
     "babel-loader": "^8.0.0",
     "copy-webpack-plugin": "^4.5.0",
     "css-loader": "^1.0.1",
+    "eslint-plugin-react-hooks": "^1.6.0",
```
Comment (skyqrose) [inline assets/package.json]: Where is this used?

**Pull request merged**